src/order/upper_lower.lean

```diff
@@ -917,6 +917,23 @@ lemma Ici_prod (x : α × β) : Ici x = (Ici x.1).prod (Ici x.2) := rfl
 @[simp] lemma prod_top (s : upper_set α) : s.prod (⊤ : upper_set β) = ⊤ := ext prod_empty
 @[simp] lemma top_prod (t : upper_set β) : (⊤ : upper_set α).prod t = ⊤ := ext empty_prod
 
+lemma upper_closure_prod_upper_closure (F₁ : set α) (F₂ : set β) :
+  (upper_closure F₁).prod (upper_closure F₂)  =
+  (⊥ : upper_set α).prod (upper_closure F₂) ⊔ (upper_closure F₁).prod (⊥ : upper_set β) :=
```

Comment:
Shouldn't that rather be `upper_closure F₁ ×ˢ upper_closure F₂`?

Reply:
Yes - if I add `infixr (name := upper_set.prod) ` ×ˢ `:82 := prod`.

Reply:
Oh yeah sure. I was mostly talking about the statement, but the notation would be nice too.

Reply:
Ah sorry, I meant that the LHS should be `upper_closure (F₁ ×ˢ F₂)`

Reply:
Okay, done.